Functions/FirebaseFunctions/FIRFunctions.m

```diff
@@ -97,33 +99,47 @@ + (instancetype)functionsForApp:(FIRApp *)app {
 }
 
 + (instancetype)functionsForRegion:(NSString *)region {
-  return [[self alloc] initWithApp:[FIRApp defaultApp] region:region];
+  return [[self alloc] initWithApp:[FIRApp defaultApp] region:region customDomain:nil];
+}
+
++ (instancetype)functionsForCustomDomain:(NSString *)customDomain {
+  return [[self alloc] initWithApp:[FIRApp defaultApp]
+                            region:kFUNDefaultRegion
+                      customDomain:customDomain];
 }
 
 + (instancetype)functionsForApp:(FIRApp *)app region:(NSString *)region {
-  return [[self alloc] initWithApp:app region:region];
+  return [[self alloc] initWithApp:app region:region customDomain:nil];
 }
 
-- (instancetype)initWithApp:(FIRApp *)app region:(NSString *)region {
++ (instancetype)functionsForApp:(FIRApp *)app customDomain:(NSString *)customDomain {
+  return [[self alloc] initWithApp:app region:kFUNDefaultRegion customDomain:customDomain];
+}
+
+- (instancetype)initWithApp:(FIRApp *)app
+                     region:(NSString *)region
+               customDomain:(NSString *)customDomain {
   return [self initWithProjectID:app.options.projectID
                           region:region
+                    customDomain:customDomain
                             auth:FIR_COMPONENT(FIRAuthInterop, app.container)
                        messaging:FIR_COMPONENT(FIRMessagingInterop, app.container)];
 }
 
 - (instancetype)initWithProjectID:(NSString *)projectID
```

Comment:
Just want to make sure, this doesn't break the public API right?

Reply:
As long as there's not a corresponding change in a public .h file